src/stub_launcher.h

```diff
@@ -145,6 +145,9 @@ class StubLauncher {
   // Kill stub process
   void KillStubProcess();
 
+  // Get a message from the stub process
+  TRITONSERVER_Error* ReceiveMessageFromStub(off_t& message);
```

Comment:
Use `bi::managed_external_buffer::handle_t` instead of `off_t`.

Reply:
Good catch. Fixed.